.github/workflows/sphinx.yml

```diff
@@ -0,0 +1,74 @@
+name: Sphinx
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+  - cron:  '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    container:
+      image: dealii/dealii:master-focal
+      options: --user root
+
+    steps:
+      - name: Setup
+        run: |
+          apt-get update
+          apt-get install -y python3.8-venv
+
+      - name: Environment
+        run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
+          python3 -m pip install sphinx
+          pip install sphinx_rtd_theme
+        shell: bash
+
+      - uses: actions/checkout@v2
+      
+      - name: Compile
+        run: |
+          mkdir build
+          cd build
+          cmake ../ -DCMAKE_BUILD_TYPE=Release
```

Comment:
Is it possible that you need to activate the env again?

Reply:
Yes, it looks like we have to do that.